modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -78,6 +78,16 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 == Additions
 {productname} 6.5 also includes the following additions:
 
+=== Support for `h` hash parameter in vimeo video url in the Media plugin.
+//#TINY-9830
+
+In previous versions of {productname}, an issue was identified that affected unlisted Vimeo videos from been added to the editor when using the media plugin.
+
+As a consequence, the media plugin failed to correctly insert the video into the content due to {productname} ignoring the "h" parameter when parsing the source URL.
+
+In {productname} 6.5, the "h" parameter is now parsed and included in the source URL by {productname}.
+
+As a result, embedding Vimeo videos into a {productname} editor using the media plugin now works as expected.
```

Comment:
```suggestion
=== Support for the `h` hash parameter in vimeo video url in the Media plugin.
//#TINY-9830

In previous versions of {productname}, an issue was identified that prevented unlisted Vimeo videos from been added to the editor when using the media plugin.

The media plugin failed to correctly insert the video into the content due to {productname} ignoring the `h` parameter when parsing the source URL.

In {productname} 6.5, the `h` parameter is now parsed and included in the source URL by {productname}.

As a result, embedding unlisted Vimeo videos into a {productname} editor using the media plugin now works as expected.
```

1. Copy-edit to sub-head to match the equivalent sub-head made to `changelog.md` and `changelog.adoc`.
2. Copy-edits to markup and text.